src/components/Extensibility/ExtensibilityWidgets.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { ExtensibilityErrBoundary } from 'components/Extensibility/ExtensibilityErrBoundary';
+import { useGetSchema } from 'hooks/useGetSchema';
+
+import { DataSourcesContextProvider } from './contexts/DataSources';
+import { useGetWidgets } from './useGetWidget';
+import { Widget } from './components-widget/Widget';
```

Comment:
shouldn't we use `components/Widget` instead? to provide all the widgets, not just the table?